docs: add security notice (#5261)
camunda/camunda-docs: docs/reference/notices.md

Camunda publishes security notices after fixes are available.

### Notice 16

#### Publication date

March 14th, 2025

#### Product affected

Camunda Zeebe

#### Impact

> Some Camunda Zeebe versions were affected by a vulnerability that allowed a malicious attacker to craft network packets that can crash Gateway.

Reviewer: Is this a gateway as defined [here](https://docs.camunda.io/docs/components/zeebe/technical-concepts/architecture/#gateways)?

Author: Yes

Reviewer: Ok, please lowercase "gateway".

```suggestion
Some Camunda Zeebe versions were affected by a vulnerability that allowed a malicious attacker to craft network packets that could crash the gateway.
```

Author: Done 👍

#### How to determine if the installation is affected

> You are using Camunda Zeebe 8.6.11

Reviewer 2: ```suggestion
You are using Camunda Zeebe 8.6.x, earlier than 8.6.12
```

Reviewer 2: And same in the versioned docs.

> You are using Camunda Zeebe 8.6.x, earlier than 8.6.12

Reviewer: I'm not sure I understand this, so I provided a suggestion. Is this for all patches between 0-12?

```suggestion
You are using Camunda Zeebe 8.6.0 - 8.6.12
```

Author: Done 👍
